File: muchos-motors-front/src/app/shared/models/InventoryCarPartLog.ts
import { CarPart } from './CarPart';
import { Customer } from './Customer';
import { OrderItem } from './OrderItem';

export class InventoryCarPartLog {
  inventoryCarPartLogId?: number | undefined;
  carPartId?: number | undefined;
  carPart?: CarPart | undefined;
  numberOfParts?: number | undefined;
  isValid?: boolean | undefined;
}

File: muchos-motors/muchos-motors-api/Models/InventoryCarPartLog.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace muchos_motors_api.Models
{
    public class InventoryCarPartLog
    {
        [Key]
        public int InventoryCarPartLogId { get; set; }

        public int InventoryLogId { get; set; }
        [ForeignKey("InventoryLogId")]
        public InventoryLog InventoryLog { get; set; }

        public int CarPartId { get; set; }
        [ForeignKey("CarPartId")]
        public CarPart CarPart { get; set; }
        public int NumberOfParts { get; set; }
        public bool IsValid { get; set; }


    }
}
